.github/workflows/pytest.yml

```diff
@@ -0,0 +1,19 @@
+name: Pytest
+on: [push, pull_request]
+jobs:
+  pytest:
+    name: pytest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      - name: Setup Poetry
+        uses: abatilo/actions-poetry@v2.1.3
+      - name: run Pytest
+        run: |
+          poetry install
+          poetry run pytest tests/crescent
```

Comment:
For tests, I think we should use a matrix to run tests on ubuntu and windows, across 3.8, 3.9, and 3.10. Example:

https://github.com/TrigonDev/apgorm/blob/7651205d8d8fcb0ee8bee5b10bbd2ee8186bf3c9/.github/workflows/ci.yml#L6-L26>\